ethereum/core/src/main/java/org/hyperledger/besu/ethereum/BlockProcessingResult.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ *  the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.hyperledger.besu.ethereum;
+
+import org.hyperledger.besu.ethereum.core.TransactionReceipt;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class BlockProcessingResult extends BlockValidationResult {
+
+  private final Optional<BlockProcessingOutputs> yield;
+  private final boolean isPartial;
```

Comment:
There are rollup usecases that will want processing results from a failed block, including only the successful results.

Reply:
only feedback here is that isPartial might imply multiple causes and messages.  non-blocking just an observation